spec/counter_culture_spec.rb

```diff
@@ -1601,16 +1606,20 @@
   end
 
   it "should use relation primary_key correctly" do
-    subcateg = Subcateg.create :subcat_id => Subcateg::SUBCAT_1
+    subcateg = Subcateg.create!
```

Comment:
Yeah… MySQL's AUTO INCREMENT seems to be more aggressive about forcing the column value to auto-increment on insert instead of allowing it to be overridden.